### PR TITLE
Creating json file

### PR DESCRIPTION
**Pull request by Manikandan-Hub**
@nishintn 
@aswathym 

Hi,
 Please find the pull request for branch **creatingJsonFile** which has the location.json file.

### CODE REVIEW

File: assets/json/location.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+[{
+
+        "Tejaswini":{"address":"<div class=\"mapInfoContent\">\
```
Comment (nishintn) [inline assets/json/location.json]: JSON are objects with key / value pairs.
eg:
{
    "type": "name",
    "properties": "value"
  }

Get rid of HTML code from it.

**Comment by nishintn** [comment]
@Manikandan-Hub ,
Please refer sample JSON structures and design one that matches your need.

**Comment by Manikandan-Hub** [comment]
@nishintn 
Hi,
Modified json file as suggested.

File: assets/json/location.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
+[{
+
+        "Tejaswini" : {
```
Comment (nishintn) [inline assets/json/location.json]: Suggest to add location name as a value in a key.

**Comment by Manikandan-Hub** [comment]
@nishintn 
Hi,
Modified json file & retrieved data for infoWindow in map.

**Pull request merged**